airflow/providers/amazon/aws/hooks/athena.py

```diff
@@ -25,7 +26,17 @@
 from airflow.providers.amazon.aws.hooks.base_aws import AwsBaseHook
 
 
-class AWSAthenaHook(AwsBaseHook):
+class AWSAthenaHook:
+    """Deprecated Hook"""
+
+    warnings.warn(
+        "This hook is deprecated. Please use `airflow.providers.amazon.aws.hooks.athena.AthenaHook`.",
+        DeprecationWarning,
+        stacklevel=2,
+    )
```

Comment:
This would make this warning unconditionally appear, and existing code to all break. You should

1. Make `AWSAthenaHook` subclass `AthenaHook` (instead of the other way around)
2. Move the deprecation warning to `__init__`.